packages/ppx_poly_record/ppx_poly_record.1.3.0/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+version: "1.3.0"
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://bitbucket.org/camlspotter/ppx_poly_record"
+bug-reports:
+  "https://bitbucket.org/camlspotter/ppx_poly_record/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ppx_poly_record"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocamlfind" {build}
```

Comment:
Obligatory comment that `ocamlfind` isn't necessary if you use dune :)

Reply:
Oops, still there...  Thanks for noticing!

Reply:
Fixed the opam file (and the template in the source)